src/components/Button/Button.js

```diff
@@ -5,9 +5,11 @@ import { applyDecorators, decoratorProp } from '../../utils';
 
 const Button = (props) => {
   const {
+    a,
```

Comment:
IMHO this is redudant. You could just assume an `a` tag if the `href` prop is present.